packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html

```diff
@@ -1,35 +1,26 @@
+		<blockquote>
+		<p>Tom,</p>
 
-	<blockquote>
-	<p>Colonel Valentin Petrov stated in 2006&nbsp;that the cosmonaut never said such words, and that the quote originated from Nikita Khrushchev&#39;s speech at the plenum of the Central Committee of the CPSU about the state&#39;s anti-religion campaign, saying:</p>
+		<p>This is just a reminder that we need the data for our Friday meeting with the external subcontractors. Can you please let me know, if all the reports are ready?</p>
 
-	<blockquote>Gagarin flew into space, but didn&#39;t see any god there.</blockquote>
+		<p>Zoe</p>
+		</blockquote>
 
-	<p>Gagarin himself was a member of the Russian Orthodox Church.</p>
-	</blockquote>
+	<p>Hi Zoe,</p>
 
-	<p>It looks like a cunning rhetoric to support an ideological point of view and you may even suppose the cosmonaut himself was not personally involved in it. Quite honestly, being a believer, he would have a hard time denying the existence of God. It is something Gherman Titov, the second Soviet cosmonaut, would rather say, being an atheist.</p>
+	<p>The reports are coming in, but the Q3 data is still missing. As far as I can tell, Shizuoka is behind the schedule with the final summary and the Expansion Department has not released their numbers yet. I will reach out to both and will keep you informed on that topic by noon, tomorrow.</p>
 
-	<p>It was, in fact, duly noted in <i>The New York Times</i>, dated May 7, 1962:</p>
-	<blockquote>
-		Maj. Gherman S. Titov, the Soviet astronaut, proclaimed his disbelief in God today. He said he saw
-		<blockquote>no God or angels</blockquote>
-		during his seventeen orbits of the earth.
+	<p>Tom</p>
 	</blockquote>
 
-	<p>A similar statement by Titov was also reported by Pavel Barashev and Yuri Dokuchayev, so it seems this was not entirely made up by Khrushchev, but rather reused and falsely attributed to the Earth's first space farer.</p>
+	<p>Please do so, this is getting urgent. Contact Haleema from ED directly. Also check Shizuoka, this is very much unlike him to leave such an important matter for the last moment.</p>
 
-	<p>Apparently there was more than one source involved in the origin of this misunderstanding. As you can see, sometimes, just like in the case of the notorious &quot;Einstein fish&quot;, things can get messed up pretty easily. And it takes a lot of time to debunk them, as there are numerous occurrences of the quote that <i>partially</i> support the case.</p>
+	<p>Hope to have this ready for review at the 2pm board meeting provided you can manage to prepare a summary and to have the full data by Thursday morning at the latest.</p>
 
-	<p>This is why you should always check your sources twice and, if necessary, quote them!</p>
+	<p>Jenna</p>
```

Comment:
```suggestion
	<p>Zoe</p>
```

:woman\_facepalming: